Xlib/protocol/rq.py

```diff
@@ -604,7 +604,8 @@ def parse_value(self, val, display):
         return self.type.parse_value(val, display)
 
     def pack_value(self, val):
-        return self.type.pack_value(val)
+        val = self.type.pack_value(val)
+        return val, len(val), None
```

Comment:
Maybe it's better to fix `pack_value` implementation at line 1074 and below? Also line 911 is interesting. For this line 607 it looks reasonable to delegate this to underlying function call. So I suggest to fix the underlying functions. What do you think?